src/server/main_service.cc

```diff
@@ -46,8 +46,7 @@ ABSL_FLAG(uint32_t, port, 6379, "Redis port");
 ABSL_FLAG(uint32_t, memcache_port, 0, "Memcached port");
 
 ABSL_FLAG(uint32_t, multi_exec_mode, 1,
-          "Set multi exec atomicity mode: 1 for global, 2 for locking ahead, 3 for locking "
-          "incrementally, 4 for non atomic");
+          "Set multi exec atomicity mode: 1 for global, 2 for locking ahead, 3 for non atomic");
```

Comment:
You also need to remove ci tests with incr mode

Reply:
We had CI tests with mode = 3, but never with 4 = non-atomic. Not sure why, I remember there were some issues with it (but I excluded them?)

I've kept it now and it seems to work (tested locally)